USER_GUIDE.md

```diff
@@ -249,4 +249,19 @@ console.log('Deleting all PITs:');
 var response = await client.deleteAllPits();
 
 console.log(response.body);
+```
+
+## Empty all pool connections
+```javascript
+console.log('Emptying all pool connections:');
```

Comment:
We should get rid of these `console.log`. It's self-explanatory what this code does, isn't it?

Reply:
> We should get rid of these `console.log`. It's self-explanatory what this code does, isn't it?

Fair enough